src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -82,7 +110,7 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			});
 			columns.push({
 				id: 'value',
-				name: localize('nodePropertyViewNameValueColumnTopHeader', "Value (Top Plan)"),
+				name: this.getPropertyViewNameValueColumnTopHeaderForOrientation(orientation),
 				field: 'value1',
```

Comment:
value1? Is that correct? And there's a value2 down below. It would be good to give these better names.